src/Movie.php

```diff
@@ -0,0 +1,29 @@
+<?php namespace Acme;
+
+use Psr\Http\Message\ResponseInterface;
+
+class Movie {
+    public string $Title, $Year, $Response;
+    public array $info = [];
+
+    public function __construct(ResponseInterface $response)
```

Comment:
you can use a static method for this instead of using the constructor

> public static function FromResponse(ResponseInterface $response) {
...
...
...

And then you call it like

> Movie::FromResponse($response)